.travis.yml

```diff
@@ -17,12 +23,12 @@ before_script:
   - CHANGED_FILES_PR=$(git diff --name-only HEAD $(git merge-base HEAD $TRAVIS_BRANCH))
 script:
   - set -o pipefail ;
-    make ci e2e status=keep deploytool=operator 2>&1 | ts '[%H:%M:%.S]' -s
+    $CMD 2>&1 | ts '[%H:%M:%.S]' -s
 after_success:
   - if [[ "${CHANGED_FILES_PR[@]}" =~ "scripts/kind-e2e/e2e.sh" ]]; then
```

Comment:
I think we should either remove this check (prefered), or additionally check when e2e libs are modified.

Reply:
We could trigger a separate job on this case, but I think it's fine for now.